counter/index.html

```diff
@@ -4,20 +4,32 @@
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>Counter app</title>
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100..900&display=swap" rel="stylesheet">
+	
 	<link rel="stylesheet" href="./styles.css">
 </head>
 <body>
 
-	<div id="counter">
-		<h1 class="counter__heading">
-			Click count:
-			<span class="counter__value">0</span>
-		</h1>
-		<div class="counter__btns">
-			<button class="counter__increment-btn btn">+</button>
-			<button class="counter__decrement-btn btn">-</button>
-		</div>
-	</div>
+		<main id="counter">
+			<h1 class="counter__heading">
+				Click count:
+				<span class="counter__value">0</span>
+			</h1>
+			<div class="counter__btns">
+				<div class="counter__btns__first-line">
```

Comment:
`counter__btns__first-line` -- у тебя сломался БЭМ. Так как у них одинаковые стили, то можно назвать их одинаково. Например, 

```suggestion
				<div class="counter__wrap">
```